File: Tsk3/JavaScript/Subtask 2/Subtask2.js
const fs = require('fs');
const path = require('path');

// Get the directory of the current script
const scriptDir = path.dirname('__filename');

// Read the content of 'input.txt'
const inputFilePath = path.join(scriptDir, 'input.txt');
const content = fs.readFileSync(inputFilePath, 'utf8');

// Write the content to 'output.txt'
const outputFilePath = path.join(scriptDir, 'output.txt');
fs.writeFileSync(outputFilePath, content);

File: Tsk3/Python/Subtask 2/Subtask2.py
import os
script_dir = os.path.dirname(os.path.abspath(__file__))
f2=open(os.path.join(script_dir,"output.txt"),"w")
f=open(os.path.join(script_dir,"input.txt"),"r")
s = f.read()
f2.write(s)
f2.close